manifests/x/XCP-ng/Center/20.04.01.33/XCP-ng.Center.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: XCP-ng.Center
+PackageVersion: 20.04.01.33
+PackageLocale: en-US
+Publisher: XCP-ng
+PackageName: XCP-ng Center 20.04.01

```

```suggestion
PackageName: XCP-ng Center
```
It's best to avoid the version number shown in the Name section of the Control Panel; so that people don't get confused with what version they're downloading.